frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -235,11 +235,7 @@
   {
     "type": "console.page/route",
     "properties": {
-      "path": [
-        "/k8s/ns/:ns/virtualmachines/:name",
-        "/k8s/ns/:ns/kubevirt.io~v1~VirtualMachine/:name",
-        "/k8s/ns/:ns/kubevirt.io~v1alpha3~VirtualMachine/:name"
-      ],
+      "path": ["/k8s/ns/:ns/virtualmachines/:name"],
```

Comment:
Is this for backwards compatibility?
All links in the app should be using the qualified reference

Reply:
its only for backwards, there is no link that is using this url in the app.
some tests however need this url